.cspell.json

```diff
@@ -302,7 +302,9 @@
     "Noto",
     "Segoe",
     "kotlinx",
-    "reauthentication"
+    "reauthentication",
```

Comment:
this is not a word, the correct compound word should be re-authentication who added this 🤦

Reply:
Removed.